cl/alerts/templates/includes/manage-alert-table-row.html

```diff
@@ -0,0 +1,32 @@
+<tr id='alert-row-{{ alert.id }}' hx-swap-oob="true">
+  <td>
+    <p class="bottom">
+      {% if alert.alert_type == "o" %}
+        Case Law
+      {% else %}
+        Oral Argument
+      {% endif %}
+    </p>
+  </td>
+  <td>
+    <p class="bottom">
+      {{ alert.name }}
```

Comment:
I think we should probably at least make this a link, like in `/profile/alerts/`. 

If we want to go wild though (and perhaps we should), could you also check how similar the two tables are, and if it makes sense to do so, merge them into a common code base? And also take note of the requested changes in this issue: https://github.com/freelawproject/courtlistener/issues/3228

Sorry, I guess this was a landmine!